src/components/main-page/Hero.tsx

```diff
@@ -0,0 +1,101 @@
+import CellImage from "@/assets/cell.png";
+import Image from "next/image";
+import { AnimatePresence, motion } from "framer-motion";
+import { useRef, useState } from "react";
+import Link from "next/link";
+
+export default function Hero() {
+  return (
+    // Landing hero
+    <div>
+      <div
+        className="flex flex-row flex-wrap items-center justify-center
```

Comment:
Might be easier to set breakpoints with explicit flex direction instead of flex wrap.

ex. Default to flex-col and switch to flex-row at breakpoint